### DISCUSSION TITLE
Does G-Helper need to be open for undervolting?

### DISCUSSION THREAD

**Discussion by christopherclarence** [seerge/g-helper#2432] [Q&A] [answered]
Hello, I'm new to G-Helper. For undervolting, does the G-Helper app need to be open? What about after a reboot?

Thank you very much for your work on the app!

**Comment by seerge** [comment, marked as answer]
@christopherclarence undervolting settings remain after app close, unless something else resets them. 

When you reboot - device resets all settings to defaults.

Why would you close app in a first place ? It's designed to stay in tray all the time.

**Reply by christopherclarence** [reply]
Just wanted to make sure how it works. Thank you for the answer!
